frontend/test/pulse/pulse.integ.spec.js

```diff
@@ -0,0 +1,161 @@
+
+import {
+    useSharedAdminLogin,
+    createTestStore,
+    createSavedQuestion
+} from "__support__/integrated_tests";
+import {
+    click,
+    setInputValue
+} from "__support__/enzyme_utils"
+
+import React from 'react';
+import { mount } from "enzyme";
+
+import {
+    CardApi,
+    PulseApi
+} from "metabase/services";
+import Question from "metabase-lib/lib/Question";
+
+import PulseListApp from "metabase/pulse/containers/PulseListApp";
+import PulseEditApp from "metabase/pulse/containers/PulseEditApp";
+import PulseListItem from "metabase/pulse/components/PulseListItem";
+import CardPicker from "metabase/pulse/components/CardPicker";
+import PulseCardPreview from "metabase/pulse/components/PulseCardPreview";
+import Toggle from "metabase/components/Toggle";
+
+import { FETCH_PULSES, SET_EDITING_PULSE, SAVE_EDITING_PULSE, FETCH_CARDS, FETCH_PULSE_CARD_PREVIEW } from "metabase/pulse/actions";
+
+describe("Pulse", () => {
+  let questionCount, questionRaw;
+  const normalFormInput = PulseApi.form_input
+
+  beforeAll(async () => {
+    useSharedAdminLogin()
+
+    const formInput = await PulseApi.form_input()
+    PulseApi.form_input = () => ({
+        channels: {
+        ...formInput.channels,
+            "email": {
+                ...formInput.channels.email,
+                "configured": true
+            }
+        }
+    })
+
+    questionCount = await createSavedQuestion(
+        Question.create({databaseId: 1, tableId: 1, metadata: null})
+            .query()
+            .addAggregation(["count"])
+            .question()
+            .setDisplay("scalar")
+            .setDisplayName("count")
+    )
+
+    questionRaw = await createSavedQuestion(
+        Question.create({databaseId: 1, tableId: 1, metadata: null})
+            .query()
+            .question()
+            .setDisplay("table")
+            .setDisplayName("table")
+    )
+
+    for (const pulse of await PulseApi.list()) {
```

Comment:
Might be nice to leave a note here about why we're deleting all pulses before this test runs.